src/scenes/PlantingSitesRouter/index.tsx

```diff
@@ -89,9 +89,10 @@ export function PlantingSitesRouter({ reloadTracking }: PlantingSitesProps): JSX
 export function PlantingSitesDraftRouter(): JSX.Element {
   return (
     <Routes>
-      <Route path={'/zone/:zoneId'} element={<PlantingSiteDraftZoneView />} />
+      <Route path={'/:plantingSiteId/zone/:zoneId'} element={<PlantingSiteDraftZoneView />} />
```

Comment:
Good catch

Reply:
Thanks for updating all the routes, that was a lot of work!